module/zfs/zio.c

```diff
@@ -167,10 +167,10 @@ zio_init(void)
 		 */
 		align = 8 * SPA_MINBLOCKSIZE;
 #else
-		if (size <= 4 * SPA_MINBLOCKSIZE) {
+		if (size < PAGESIZE) {
```

Comment:
This will result in 3 additional caches being created for tiny buffers.  That's not a problem and likely desirable but worth mentioning.